tests/run-openzep.sh

```diff
@@ -9,11 +9,24 @@ PORT=8545
 
 while (netcat -z 127.0.0.1 "$PORT") ; do sleep 0.1; done
 
-./kevm web3-ganache "$PORT" --shutdownable &
+./kevm web3 "$PORT" --shutdownable &
 kevm_client_pid="$!"
 
 while (! netcat -z 127.0.0.1 "$PORT") ; do sleep 0.1; done
 
+./kevm web3-send "$PORT" firefly_addAccount '{"key":"0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501200", "balance":"0xd3c21bcecceda1000000"}'
+./kevm web3-send "$PORT" firefly_addAccount '{"key":"0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501201", "balance":"0xd3c21bcecceda1000000"}'
+./kevm web3-send "$PORT" firefly_addAccount '{"key":"0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501202", "balance":"0xd3c21bcecceda1000000"}'
+./kevm web3-send "$PORT" firefly_addAccount '{"key":"0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501203", "balance":"0xd3c21bcecceda1000000"}'
+./kevm web3-send "$PORT" firefly_addAccount '{"key":"0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501204", "balance":"0xd3c21bcecceda1000000"}'
+./kevm web3-send "$PORT" firefly_addAccount '{"key":"0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501205", "balance":"0xd3c21bcecceda1000000"}'
+./kevm web3-send "$PORT" firefly_addAccount '{"key":"0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501206", "balance":"0xd3c21bcecceda1000000"}'
+./kevm web3-send "$PORT" firefly_addAccount '{"key":"0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501207", "balance":"0xd3c21bcecceda1000000"}'
+./kevm web3-send "$PORT" firefly_addAccount '{"key":"0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501208", "balance":"0xd3c21bcecceda1000000"}'
+./kevm web3-send "$PORT" firefly_addAccount '{"key":"0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501209", "balance":"0xd3c21bcecceda1000000"}'
+./kevm web3-send "$PORT" firefly_setGasLimit '"0xfffffffffff"'
+./kevm web3-send "$PORT" firefly_genesisBlock
```

Comment:
Is this specific to OpenZeppelin or should `./kevm web3-ganache` always do this?

Reply:
It's openzeppelin specific. web3-ganache replicates Ganache's default behavior.